Aptoide/aptoide-client-v8
# Feature/mob 908 work manager campaign notifications

**What does this PR do?**

This PR changes the way campaign notifications are handled; now using workManager;

**Database changed?**

No

**Where should the reviewer start?**

- [ ] AlarmSyncScheduler.java
- [ ] CampaignNotificationWorker.kt

**How should this be manually tested?**

I changed the network call in the code to send a dummy notification instead of going to the webservice. You can also rewrite the charles request if you want to try a different method; I also recommend changing the gradle properties notification alarm time to 20 minutes instead of 6 hours (the minimum time for workManager is 15m, I chose 20 to make sure I didn't run into any issues there);

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-908](https://aptoide.atlassian.net/browse/MOB-908)

**Code Review Checklist**
